src/libraries/Microsoft.Extensions.Hosting.Systemd/src/Microsoft.Extensions.Hosting.Systemd.csproj

```diff
@@ -8,6 +8,8 @@
     <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>true</IsPackable>
+    <!-- CS8981: The type name 'libc' only contains lower-cased ascii characters. Such names may become reserved for the language. -->
+    <NoWarn>$(NoWarn);CS8981</NoWarn>
```

Comment:
I believe elsewhere we ended up changing the casing to Libc as part of changes @jcouv made for this warning.  This library may have escaped his notice.

Reply:
Although I see a bunch of other warnings being issued, for things like procfs.  @jcouv, did we just punt on those?  Maybe we should suppress this warning globally for dotnet/runtime, since it's not particularly friendly to the unix way of naming libs?

Reply:
(If we do suppress it everywhere, we should change any Libc made as part of @jcouv's changes back to libc.)

Reply:
I thought I'd caught all lower-cased types names previously ([PR](https://github.com/dotnet/runtime/pull/62507)). Not sure how I missed some. I'd used `build.cmd libs+libs.tests` for my local build  (as [documented](https://github.com/dotnet/roslyn/blob/main/docs/contributing/Building%2C%20Debugging%2C%20and%20Testing%20on%20Windows.md#testing-on-the-dotnetruntime-repo) on roslyn side).
In that PR, the resolution was to either change casing or to add an `@` character to escape the name. I would recommend sticking with those resolutions, rather than suppressing the diagnostic.

Reply:
You probably missed the OS-specific configurations. You would have needed to do an AllConfigurations build to make sure you hit all the managed code.

Reply:
I've updated `libc` and `libproc` to escape with `@` similarly to `procfs` for consistency.